src/components/Map.js

```diff
@@ -38,15 +40,14 @@ export const Map = ({
   const handleCenterMoved = (event) => {
     const lat = event?.center?.lat();
     const lng = event?.center?.lng();
-    const newMapCenter = { lat, lng };
-    setNewCenterMap(newMapCenter);
+    setNewCenterMap({ lat, lng, isDragged: true });
   };
 
   return (
     <div className="map">
       <GoogleMapReact
         bootstrapURLKeys={{
-          key: 'AIzaSyBVolVQlm384nMSDHcDOVLfyqrahoPVYl4',
+          key: 'AIzaSyCbfV0IAdkkGv-9mmuAkUJNzCPPfGRO6v0',
```

Comment:
@katia-utochkina & @jssckbl: I suggest before you will merge with `main` your PR, **change** to another API key (because this token is not enabled **Places API**). Please.

Reply:
@alxmcr and @segdeha, for some reason,  ``AIzaSyBVolVQlm384nMSDHcDOVLfyqrahoPVYl4`` is not working here, and I have no idea why. The map gets displayed only when this key is used  ``AIzaSyCbfV0IAdkkGv-9mmuAkUJNzCPPfGRO6v0``

![1](https://user-images.githubusercontent.com/16920088/130193094-62ccbab5-0683-45e2-9f2d-8d2e48048f09.png)

Reply:
@katia-utochkina: Andrew gave us a new API token because the last token wasn't enabled by the Places API. However, it can't use with `localhost` only with `*.web.app` (or a link generated by GitHub). It's a reason why you couldn't review our PR in your local machines only with the link generated (but the code you could review it). Before you will merge your PR with `main`. I suggest changing the `API key` and checking **Google Maps & Place API** works!. Please.

Reply:
@alxmcr, got it, thank you  for clarification